modules/smart-agent_systemd-timers/conf/readme.yaml

```diff
@@ -0,0 +1,8 @@
+documentations:
+  - name: Smart Agent monitor collectd/systemd
+    url: https://docs.signalfx.com/en/latest/integrations/agent/monitors/collectd-systemd.html
+
+source_doc: |
+  This module checks Systemd services launched by Systemd timers. The difference with [smart-agent_systemd-services](https://github.com/claranet/terraform-signalfx-detectors/tree/master/modules/smart-agent_systemd-services) is that services launched by timers are not supposed to be always running, and instead can be stopped most of the time.
+
+  Detectors are designed to check that the job succeeded, the service have not been removed, and the job have been launched during the past 30 minutes by default (you can override this behavior with the `timer_has_not_run_for_lasting_duration_major` variable).
```

Comment:
```suggestion
  Detectors are designed to check that the job succeeded, the service have not been removed, and the job have been launched during the past 30 minutes by default (you can override this behavior with the `execution_delay_lasting_duration_major` variable).
```

Reply:
Should be better now. Also, I can confirm both detectors can be used together, of course you will most likely not monitor the same services with both of them, but if you use `filtering_custom` this is not a problem.

Reply:
@xp-1000 I cleaned up history, I let you confirm everything looks good to you now :)

Reply:
hi @NSenaud it looks good to me, just 2 minor comments.